api/src/pcapi/core/users/api.py

```diff
@@ -1790,3 +1791,46 @@ def enable_new_pro_nav(user: models.User) -> None:
     pro_new_nav_state.newNavDate = datetime.datetime.utcnow()
     db.session.add(pro_new_nav_state)
     db.session.commit()
+
+
+def clean_gdpr_extracts() -> None:
+    extension = "zip"
+    folder = settings.GCP_GDPR_EXTRACT_FOLDER
+    files = object_storage.list_files(
+        folder=folder,
+        bucket=settings.GCP_GDPR_EXTRACT_BUCKET,
+    )
+    files_ids = set()
+    for file_path in files:
+        try:
+            file_name = file_path.split("/")[-1]
+            extract_id = int(file_name[: -(len(extension) + 1)])
```

Comment:
Il y a des modules qui font ce genre de choses très bien :slightly_smiling_face: 
```suggestion
            from pathlib import Path
            extract_id = int(Path(file_path).stem)
```
(pathlib est déjà utilisé dans api)